src/test/java/seedu/address/logic/commands/FindByWorkExperienceCommandTest.java

```diff
@@ -71,11 +67,13 @@ public void execute_zeroKeywords_noPersonFound() {
     @Test
     public void execute_multipleKeywords_multiplePersonsFound() {
         String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 2);
-        WorkExperienceContainsKeywordsPredicate predicate = new WorkExperienceContainsKeywordsPredicate("Intern", "Google", "2024");
+        WorkExperienceContainsKeywordsPredicate predicate =
```

Comment:
The modification of WorkExperienceContainsKeywordsPredicate is a solid update that better reflects the use case by including the additional keywords Intern, Google, 2024. This makes the search functionality more flexible. Ensure that the keywords being tested align with the team’s intended use cases for future enhancements or additional edge cases.

Reply:
The update to WorkExperienceContainsKeywordsPredicate is a good improvement, as it now supports additional keywords like "Intern," "Google," and "2024," which enhances the flexibility of the search functionality. Just make sure that the tested keywords are in line with the team's intended use cases, especially for any future updates or edge cases.